### DISCUSSION TITLE
Controlling page height in FlexibleColumnLayout

### DISCUSSION THREAD

**Discussion by ozreads** [UI5/webcomponents#6717] [Q&A]
Hi.
First thanks for the great work. 

I have a problem with FlexibleColumnLayout. I am trying to build a page where I have two columns, but they take the full height of the screen. I tried to have a parent div and the FlexibleColumnLayout as a child and I set the parent div to 100vh min-height, but no luck. I tried to apply 100vh min-height directly on FlexibleColumnLayout but also no luck. I always end up with the screenshot below.
![image](https://user-images.githubusercontent.com/115879540/225404413-4c470762-e655-4637-bf3b-55fa39da9c2e.png)

My goal is to acheive something like this:
![image](https://user-images.githubusercontent.com/115879540/225405697-72d25c3c-2c74-42e3-9fa7-24c9d8ed446b.png)

Sadly, the only way for me to achieve this is by setting a min-height on each column which is not ideal because each column might have more or less content than the other. Here is how my component looks, but without any height.\

```
const Layout = () => {
  return (
    <FlexibleColumnLayout
      layout={"TwoColumnsStartExpanded"}
      startColumn={
        <div
          style={{
            backgroundColor: "red",
          }}
        >
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing elit.
            Dignissimos, pariatur provident quibusdam voluptate nam aut
            praesentium eligendi distinctio neque fugit quas, id accusantium
            quisquam excepturi corporis placeat quasi! Nemo, cumque.
          </p>
        </div>
      }
      midColumn={
        <div
          style={{
            backgroundColor: "yellow",
          }}
        >
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing elit.
            Dignissimos, pariatur provident quibusdam voluptate nam aut
            praesentium eligendi distinctio neque fugit quas, id accusantium
            quisquam excepturi corporis placeat quasi! Nemo, cumque.
          </p>
        </div>
      }
    />
  );
};
 
```

**Comment by Lukas742** [comment]
Hi @ozreads 

since this component is developed by the ui5-webcomponents team, I'm going to forward this question to their repo. 

___

Hi colleagues, 

here you can find the example given by @ozreads in plain html+ui5wc: https://codesandbox.io/s/ui5-webcomponents-forked-2dvvrb?file=/index.html

**Reply by ozreads** [reply]
Thanks
